src/Umbraco.Tests.Common/Builders/Extensions/ContentTypeBuilderExtensions.cs

```diff
@@ -0,0 +1,143 @@
+                        .WithSortOrder(1)
+                        .WithDataTypeId(-88)
+                        .Done()
+                    .Done()
+                .AddAllowedTemplate()
+                    .WithId(200)
+                    .WithAlias("textPage")
+                    .WithName("Text Page")
+                    .Done()
+                .AddAllowedTemplate()
+                    .WithId(201)
+                    .WithAlias("textPage2")
+                    .WithName("Text Page 2")
+                    .Done()
+                .WithDefaultTemplateId(200)
+                .AddAllowedContentType()
+                    .WithId(888)
+                    .WithAlias("sub")
+                    .WithSortOrder(8)
+                    .Done()
+                .AddAllowedContentType()
+                    .WithId(889)
+                    .WithAlias("sub2")
+                    .WithSortOrder(9)
+                    .Done()
+                .Build();
+        }
+
+        public static MediaType BuildImageMediaType(this MediaTypeBuilder builder)
+        {
+            return (MediaType)builder
+                .WithId(10)
+                .WithAlias(Constants.Conventions.MediaTypes.Image)
+                .WithName("Image")
+                .WithCreatorId(22)
+                .WithDescription("test")
+                .WithIcon("icon")
+                .WithThumbnail("thumb")
+                .WithSortOrder(5)
+                .WithLevel(3)
+                .WithPath("-1,4,10")
+                .WithPropertyTypeIdsIncrementingFrom(200)
+                .WithMediaPropertyGroup()
+                .Build();
+        }
+
+        public static MemberType BuildSimpleMemberType(this MemberTypeBuilder builder)
+        {
+            return (MemberType)builder
+                .WithId(10)
+                .WithAlias("memberType")
+                .WithName("Member type")
+                .WithCreatorId(22)
+                .WithDescription("test")
+                .WithIcon("icon")
+                .WithThumbnail("thumb")
+                .WithSortOrder(5)
+                .WithLevel(3)
+                .WithPath("-1,4,10")
+                .WithPropertyTypeIdsIncrementingFrom(200)
+                .AddPropertyGroup()
+                    .WithName("Content")
+                    .AddPropertyType()
+                        .WithPropertyEditorAlias(Constants.PropertyEditors.Aliases.TextBox)
+                        .WithValueStorageType(ValueStorageType.Nvarchar)
+                        .WithAlias("title")
+                        .WithName("Title")
+                        .WithSortOrder(1)
+                        .WithDataTypeId(-88)
+                        .Done()
+                    .AddPropertyType()
+                        .WithPropertyEditorAlias(Constants.PropertyEditors.Aliases.TextBox)
+                        .WithValueStorageType(ValueStorageType.Ntext)
+                        .WithAlias("bodyText")
```

Comment:
Why do we need to specific this explicit, and not use the default values?

Reply:
Have made some updates to address this.